### DISCUSSION TITLE
Status of mail when using smtp relay

### DISCUSSION THREAD

**Discussion by skowrons** [postalserver/postal#2279] [Help with using Postal]
Hello,

we have the following flow:

client ----> postal ----> mailcow ----> goal

Postal is configured to use mailcow as a smtp relay. But know the status feature (sent, soft fail, ...) isn't working anymore. Are there any additonal configuration that needed to be done?

**Comment by willpower232** [comment]
I don't know if this is possible but I also don't know how Postal helps you if you already have mailcow?
